src/firmware/boards/robot_stm32h7/lwip.c

```diff
@@ -91,7 +91,7 @@ void MX_LWIP_Init(void)
     memset(&attributes, 0x0, sizeof(osThreadAttr_t));
     attributes.name       = "EthLink";
     attributes.stack_size = INTERFACE_THREAD_STACK_SIZE;
-    attributes.priority   = osPriorityBelowNormal;
+    attributes.priority   = osPriorityNormal3;
```

Comment:
any reason why normal3?

Reply:
i was just messing around with priorities when I thought that was the reason why ethernet wasn't working

but i've put it back, it doesn't matter